src/app/core/views/dashboard/dashboard.component.html

```diff
@@ -1,14 +1,15 @@
 <div class="container-fluid">
   <div class="row">
     <div class="col-md-3">
-      <div *ngIf="user$ | async as user">
-        <img class="img-responsive" src="{{ user.photoURL }}" alt="">
+      <div class="user-info" *ngIf="user$ | async as user">
```

Comment:
Mae siempre hay que dejar el ngIf como primer atributo del tag para que si es false, que se descarte antes de agregar cualquier atributo.
Despues lo acomodamos. Not a big deal!